invokeai/app/services/config.py

```diff
@@ -426,6 +430,9 @@ class InvokeAIAppConfig(InvokeAISettings):
     version             : bool = Field(default=False, description="Show InvokeAI version and exit", category="Other")
     # fmt: on
 
+    class Config:
+        validate_assignment = True
+
```

Comment:
This serves to parse & validate mutations to the instance of a model. In this case, it effectively assigns the default value to missing fields. Keep in mind this means we get an exception when any assignment value is malformed.

I *think* this is for the best.

Reply:
Gotcha. This is only needed bc we are using `setattr()` instead of just instantiating the class all at once. 

But if we are mutating the class at runtime, this should be turned on regardless of how we create an `InvokeAIAppConfig`, as a safety measure.